server/services/telegram/lib/message.new.js

```diff
@@ -30,7 +30,7 @@ async function newMessage(msg) {
     user_id: user.id,
     user,
     language: user.language,
-    date: msg.date,
+    created_at: new Date(msg.date * 1000).toISOString(),
     text: msg.text,
```

Comment:
Now you require the `message.create`  function to have an `id`, but here there is no id passed. 

It can be ok, but it should be clear. Where is the id generated now? does it work? (I guess yes, and the id is generated by sequelize, but the code is not that clear on that point)

Reply:
I generated one UUIDv4, more clear in code.